sevntu-checks/pom.xml

```diff
@@ -216,6 +216,9 @@
               <regex><pattern>.*.checks.coding.NoNullForCollectionReturnCheck</pattern><branchRate>85</branchRate><lineRate>96</lineRate></regex>
               <regex><pattern>.*.checks.coding.OverridableMethodInConstructorCheck</pattern><branchRate>88</branchRate><lineRate>99</lineRate></regex>
               <regex><pattern>.*.checks.design.HideUtilityClassConstructorCheck</pattern><branchRate>94</branchRate><lineRate>100</lineRate></regex>
+              <regex><pattern>.*.checks.coding.ForbidCertainMethodsLoader</pattern><branchRate>66</branchRate><lineRate>56</lineRate></regex>
+              <regex><pattern>.*.checks.coding.ForbidCertainMethodsCheck</pattern><branchRate>66</branchRate><lineRate>90</lineRate></regex>
+              <regex><pattern>.*.checks.coding.ForbidCertainMethodsCheck\$CallInfo</pattern><branchRate>100</branchRate><lineRate>88</lineRate></regex>
```

Comment:
We encourage you to strive for 100%.
We have started this with issue #520 .

Reply:
I will add more tests.

Reply:
Done.